samples/fastlane/Fastfile

```diff
@@ -57,5 +57,46 @@ platform :android do
       groups: test_groups,
       release_notes: build_notes
     )
-  end 
+  end
+
+  lane :build_sample_app_for_sdk_release do |values|
```

Comment:
I opted for creating a new lane since the existing one is already fairly complicated. Also if we decide to move to a single app, we can just deprecate the older lane.

Reply:
Hmm, I'm unsure about this because again, both lanes have 70% of the same code, apart from some hardcoded values in this lane. I don't mind separate lane, but then could we probably move the shared code in methods that both lanes can utilize? 

Also, could you point out the complicated parts in the other lane so we can see how we can make them less complicated in future? 

for example, something like this, ensure, that when you are testing a feature branch we don't show the version of the SDK in user agent, but rather the branch its being tested on. 

```
    UI.important("Updating the SDK's source code version to non-production version. This allows the sample apps to show the SDK version at runtime for app user to better understand the version of the SDK they are running.")
    sh("../../../scripts/update-version.sh \"#{new_app_version}.#{new_build_number}\"")
```